src/zoid/buttons/util.js

```diff
@@ -126,16 +126,11 @@ export function getVenmoEligibility(): EligibilityExperiment {
 
   const isVenmoFundingEnabled =
     enableFunding && enableFunding.indexOf(FUNDING.VENMO) !== -1;
-  const isNativeSupported = isSupportedNativeBrowser();
```

Comment:
Non-supported browsers and WebView handling will be driven in scnw flow experience decisions.